content/en/docs/User guides/submitting-jobs.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Submitting jobs"
+linkTitle: "Submitting jobs"
+date: 2019-03-15
+weight: 1
+description: >
+  Submit rendering jobs to OpenCue from CueSubmit
+---
+
+This guide describes how to submit a job to OpenCue using the stand-alone
+version of CueSubmit and configure relevant rendering settings. A job is a
+collection of layers, which is sent as a script to the queue to be processed
+on remote cores. Each job can contain one or more layers, which are sub-jobs
+in an outline script job. Each layer contains a frame range and a command to
+execute.
+
+## Before you begin
+
+Make sure your OpenCue admin has
+[installed CueSubmit](/docs/getting-started/installing-cuesubmit/).
+
+## Submitting a job
+
+To submit a job to OpenCue:
+
+1.  Start CueSubmit.
+
+    {{% alert title="Note" color="info"%}}The instructions for running
+    CueSubmit vary depending on the installation method. To learn more, see
+    [Installing CueSubmit](/docs/getting-started/installing-cuesubmit)
+    or contact your OpenCue admin.{{% /alert %}}
+
+1.  Enter a **Job Name**.
+
+    The job name is an arbitrary value that you choose when creating the
+    job.
+    
+    {{% alert title="Note" color="info"%}}You can follow the naming convention
+    for your rendering facility, as long as the job name is unique, contains
+    more than 3 characters, and contains no spaces.{{% /alert %}}
+
+1.  Select a **Show**.
+
+    A show is a group of related jobs to be processsed. Jobs you submit to
+    OpenCue exist within a show.
+
+1.  Enter a **Shot**, which is a series of uninterupted frames you need to
+    render.
+
+1.  Enter a **Layer Name** to name the first layer in your job.
```

Comment:
For shot and layer name - one thing I noticed this past week demoing OpenCue to potential users is that they're confused about what values they should enter here.

Maybe we should specify that they are arbitrary strings? I.e. "shot" should be the name of the shot but _can_ be anything you want? And layer name can be whatever you want, but you should pick something descriptive based on what the layer is doing, such as "render" or "compositing".

Reply:
Great feedback, thanks. I was starting to think the same thing as I've been testing OpenCue myself more rigorously to prepare to demo it.

I've updated the doc.